forge-game/src/main/java/forge/game/staticability/StaticAbility.java

```diff
@@ -331,6 +331,8 @@ public final boolean applyAbility(final String mode, final Card card, final Game
             return StaticAbilityCantAttackBlock.applyCantBlockByAbility(this, card, (Card)target);
         } else if (mode.equals("CanAttackIfHaste")) {
             return StaticAbilityCantAttackBlock.applyCanAttackHasteAbility(this, card, target);
+        } else if (mode.equals("CanAttackDefender")) {
+            return StaticAbilityCanAttackDefender.applyCanAttackAbility(this, card, target);
```

Comment:
better move that part to StaticAbilityCanAttackDefender like i did with some other abilities so i can remove this ambiguous "applyAbility" in the future